src/style.scss

```diff
@@ -196,12 +198,26 @@ input,
 
   .public-key {
     margin-bottom: 1em;
-    code {
-      overflow: hidden;
-      text-overflow: ellipsis;
-      margin-right: 0.5em;
-    }
+    background-color: $color-primary;
+    border: 0px solid;
+    border-radius: 0.3em;
+    display: flex;
   }
+
+  code {
+    margin-right: 0.5em;
+  }
```

Comment:
I would put this inside `.public-key` since it's a specific style condition when displaying that field.